### PR TITLE
Gradual rollout updates

### PR DESCRIPTION
**Pull request by WalshyDev**
### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Review by irvinebroque — COMMENTED** [review]
We will need to be clear that in order to use the command without `--x-versions`, you need to use a version of Wrangler after <version>

File: src/content/docs/workers/wrangler/commands.mdx [hunk 1]
```diff
@@ -2268,24 +2252,18 @@ wrangler versions secret bulk <FILENAME> [OPTIONS] --experimental-versions
 
 This command is currently in closed beta. Report bugs in [GitHub](https://github.com/cloudflare/workers-sdk/issues/new/choose).
```
Comment (tanushree-sharma) [inline src/content/docs/workers/wrangler/commands.mdx]:
```suggestion
This command is currently in closed beta. Report bugs in [GitHub](https://github.com/cloudflare/workers-sdk/issues/new/choose).
```

File: src/content/docs/workers/wrangler/commands.mdx [hunk 2]
```diff
@@ -2332,24 +2310,17 @@ wrangler deployments list [OPTIONS]
...
+The minimum required wrangler version to use these commands is 3.40.0. For versions before XX (TODO), you will need to add the `--x-versions` flag.
 :::
 
-### `list --experimental-versions`
+### `list`
 
 Retrieve details for the 10 most recent [deployments](/workers/configuration/versions-and-deployments/#deployments). Details include `Created on`, `Author`, `Source`, an optional `Message`, and metadata about the `Version(s)` in the deployment.
 
 ```txt
-wrangler deployments list [OPTIONS] --experimental-versions
```
Comment (tanushree-sharma) [inline src/content/docs/workers/wrangler/commands.mdx:2348]: `list` is duplicated. We should delete the first one (this one was previous to us launching versions) and move the note down to the second

File: src/content/docs/workers/wrangler/commands.mdx [hunk 3]
```diff
@@ -2398,14 +2359,12 @@ wrangler rollback [<DEPLOYMENT_ID>] [OPTIONS]
 - `--message` string optional
   - Add message for rollback. Accepts empty string. When specified, interactive prompts for rollback confirmation and message are skipped.
 
-### `rollback --experimental-versions`
```
Comment (tanushree-sharma) [inline src/content/docs/workers/wrangler/commands.mdx:2403]: Double rollback commands. This one should stay, top one can be deleted

**Comment by tanushree-sharma** [comment]
Need to remove the `wrangler deployments view` command

File: src/content/docs/workers/configuration/versions-and-deployments/index.mdx [hunk 4]
```diff
@@ -64,20 +47,22 @@ Changes uploaded with [`wrangler deploy`](/workers/wrangler/commands/#deploy), v
...
+:::note
+When using [wrangler](/workers/wrangler/), changes made to a Worker's triggers [routes,  domains](/workers/configuration/routing/) or [cron triggers](/workers/configuration/cron-triggers/) need to be applied with the command [`wrangler triggers deploy](/workers/wrangler/commands/#triggers).
+:::
+
+:::note
+New versions are not created when you make changes to [resources connected to your Worker](/workers/runtime-apis/bindings/). For example, if two Workers (Worker A and Worker B) are connected via a [service binding](/workers/runtime-apis/bindings/service-bindings/), changing the code of Worker B will not create a new version of Worker A. Changing the code of Worker B will only create a new version of Worker B. Changes to the service binding (such as, deleting the binding or updating the [environment](/workers/wrangler/environments/) it points to) on Worker A will also not create a new version of Worker B.
```
[the remark this answers is not in the document]
Comment (WalshyDev) [reply src/content/docs/workers/configuration/versions-and-deployments/index.mdx:65]: @kodster28 is there a way to disable hyperlink firing on links?

File: src/content/docs/workers/wrangler/commands.mdx (hunk 3 above)
Comment (WalshyDev) [reply src/content/docs/workers/wrangler/commands.mdx:2403]: Do we want to move the caution down?
> A rollback will immediately create a new deployment with the specified version of your Worker and become the active deployment across all your deployed routes and domains.

I think this is pretty understood anyway but happy to move down

File: src/content/docs/workers/configuration/versions-and-deployments/index.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (kodster28) [reply src/content/docs/workers/configuration/versions-and-deployments/index.mdx:65]: Ah, damn, this is an edge case I missed previously.

Let me fix this

File: src/content/docs/workers/configuration/versions-and-deployments/index.mdx [hunk 5]
```diff
@@ -98,11 +83,11 @@ To view your deployments in the Cloudflare dashboard:
 
 ### First upload
 
-You must use [C3](/workers/get-started/guide/#1-create-a-new-worker-project) or [`wrangler deploy`](/workers/wrangler/commands/#deploy) the first time you create a new Workers project. Using [`wrangler versions upload --experimental-versions`](/workers/wrangler/commands/#upload) the first time you upload a Worker will fail.
+You must use [C3](/workers/get-started/guide/#1-create-a-new-worker-project) or [`wrangler deploy`](/workers/wrangler/commands/#deploy) the first time you create a new Workers project. Using [`wrangler versions upload](/workers/wrangler/commands/#upload) the first time you upload a Worker will fail.
```
[the remark this answers is not in the document]
Comment (kodster28) [reply src/content/docs/workers/configuration/versions-and-deployments/index.mdx]: As is this, fun.

Comment (kodster28) [inline src/content/docs/workers/configuration/versions-and-deployments/index.mdx]:
```suggestion
You must use [C3](/workers/get-started/guide/#1-create-a-new-worker-project) or [`wrangler deploy`](/workers/wrangler/commands/#deploy) the first time you create a new Workers project. Using [`wrangler versions upload`](/workers/wrangler/commands/#upload) the first time you upload a Worker will fail.
```

File: src/content/docs/workers/configuration/versions-and-deployments/index.mdx [hunk 6]
```diff
@@ -98,11 +83,11 @@ To view your deployments in the Cloudflare dashboard:
...
 
 ### Service worker syntax
 
-Service worker syntax is not supported for versions that are uploaded through [`wrangler versions upload --experimental-versions`](/workers/wrangler/commands/#upload). You must use ES modules format.
+Service worker syntax is not supported for versions that are uploaded through [`wrangler versions upload](/workers/wrangler/commands/#upload). You must use ES modules format.
```
Comment (kodster28) [inline src/content/docs/workers/configuration/versions-and-deployments/index.mdx]:
```suggestion
Service worker syntax is not supported for versions that are uploaded through [`wrangler versions upload`](/workers/wrangler/commands/#upload). You must use ES modules format.
```

**Review by kodster28 — COMMENTED** [review]
I think these bits are what's tripping up the linting

File: src/content/docs/workers/configuration/versions-and-deployments/index.mdx (hunk 5 above)
Comment (WalshyDev) [reply src/content/docs/workers/configuration/versions-and-deployments/index.mdx]: ohhhh damn it good catch haha

File: src/content/docs/workers/wrangler/commands.mdx (hunk 3 above)
Comment (tanushree-sharma) [reply src/content/docs/workers/wrangler/commands.mdx:2403]: Let's keep it? doesnt hurt to

**Comment by tanushree-sharma** [comment]
Was it intentional to move rollback into the deployments top level command? Should be represented as a top level command in the docs

**Comment by WalshyDev** [comment]
Nope, I just removed the top level when removing the dupe - fixed

File: src/content/docs/workers/wrangler/commands.mdx [hunk 7]
```diff
@@ -2153,24 +2153,18 @@ wrangler whoami
 
 [Versions](/workers/configuration/versions-and-deployments/#versions) are currently in beta. Report bugs in [GitHub](https://github.com/cloudflare/workers-sdk/issues/new/choose).
```
Comment (tanushree-sharma) [inline src/content/docs/workers/wrangler/commands.mdx]:
```suggestion
```

File: src/content/docs/workers/wrangler/commands.mdx [hunk 8]
```diff
@@ -2270,88 +2254,39 @@ wrangler versions secret bulk <FILENAME> [OPTIONS] --experimental-versions
 
 This command is currently in closed beta. Report bugs in [GitHub](https://github.com/cloudflare/workers-sdk/issues/new/choose).
```
Comment (tanushree-sharma) [inline src/content/docs/workers/wrangler/commands.mdx]:
```suggestion
```

File: src/content/docs/workers/wrangler/commands.mdx [hunk 9]
```diff
@@ -2360,54 +2295,22 @@ wrangler deployments list [OPTIONS] --experimental-versions
...
-:::note
-
-[Rollbacks](/workers/configuration/versions-and-deployments/rollbacks/) is currently in open beta. Report bugs in [GitHub](https://github.com/cloudflare/workers-sdk/issues/new/choose).
-
-:::
-
-[Roll back](/workers/configuration/versions-and-deployments/rollbacks/) to a specified [version](/workers/configuration/versions-and-deployments/#versions) by ID, or to the previous version if no ID is provided. The `rollback` command will prompt you for confirmation of the rollback. On confirmation, you will be prompted to provide an optional rollback message.
-
-There are limitations on which versions you can rollback to. Refer to [Rollbacks Limits](/workers/configuration/versions-and-deployments/rollbacks/#limits) for more information.
+## `rollback
```
Comment (tanushree-sharma) [inline src/content/docs/workers/wrangler/commands.mdx]:
```suggestion
## `rollback`
```

**Comment by tanushree-sharma** [comment]
couldn't pinpoint exactly where in the file was causing this but looks like the note is bleeding down: 
image

File: src/content/docs/workers/wrangler/commands.mdx [hunk 10]
```diff
@@ -2360,54 +2295,22 @@ wrangler deployments list [OPTIONS] --experimental-versions
...
-
-- `DEPLOYMENT_ID` string optional
-  - The ID of the deployment you wish to roll back to.
-- `--name` string optional
-  - Perform on a specific Worker rather than inheriting from `wrangler.toml`.
-- `--message` string optional
-  - Add message for rollback. Accepts empty string. When specified, interactive prompts for rollback confirmation and message are skipped.
-
-### `rollback --experimental-versions`
+:::`
```
Comment (WalshyDev) [inline src/content/docs/workers/wrangler/commands.mdx]:
```suggestion
:::
```

**Review by tanushree-sharma — APPROVED** [review]

File: src/content/docs/workers/configuration/versions-and-deployments/index.mdx [hunk 11]
```diff
@@ -64,20 +47,22 @@ Changes uploaded with [`wrangler deploy`](/workers/wrangler/commands/#deploy), v
...
-
 :::note
 
+Wrangler versions before 3.73.0 require you to specify a `--x-versions` flag.
 
-When using [wrangler](/workers/wrangler/), changes made to a Worker's triggers [routes,  domains](/workers/configuration/routing/) or [cron triggers](/workers/configuration/cron-triggers/) need to be applied with the command [`wrangler triggers deploy --experimental-versions`](/workers/wrangler/commands/#triggers). 
 :::
 
-:::note
+To create a new version of your Worker that is not deployed immediately, use the [`wrangler versions upload`](/workers/wrangler/commands/#upload) command or create a new version via the Cloudflare dashboard using the **Save** button. You can find the **Save** option under the down arrow beside the "Deploy" button.
```
Comment (dcpena) [inline src/content/docs/workers/configuration/versions-and-deployments/index.mdx:56]: I don't want to block this for a single comment, but is the Save option difficult to locate? Typically, we try to refrain from adding directions for locating a button as they relate to other options on the page in the event the UI changes. If it's pretty easy to find, I think we can scrap the last the last sentence. :)

**Review by dcpena — APPROVED** [review]

File: src/content/docs/workers/configuration/versions-and-deployments/index.mdx (hunk 11 above)
Comment (WalshyDev) [reply src/content/docs/workers/configuration/versions-and-deployments/index.mdx:56]: It's still pretty new and not super obvious for those used to the dash. I think keeping it for now makes sense but probably good to remove later

**Pull request merged**